### ISSUE TITLE
USWDS-Site - Bug: Release Notes not rendering

### ISSUE THREAD

**Issue by DaveSkender** [uswds/uswds-site#2136]
### Describe the bug

Looks like the markdown is supposed to render as HTML here, but is not.

https://designsystem.digital.gov/about/releases/

### Steps to reproduce the bug

See https://designsystem.digital.gov/about/releases/

### Expected Behavior

Legible release notes

### Screenshots

![IMG_0074](https://github.com/uswds/uswds-site/assets/8432125/c34775f2-ce72-4a56-a69b-035f28c3802a)

### System setup

iPad mini, Safari

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by danbrady** [comment]
I dived into this to see what's going on. Here's what I discovered:
- Each release note has its own HTML heading structure (starting at H1).
- They are then injected into the doc site from [github releases api](https://api.github.com/repos/uswds/uswds/releases)
- The doc page, naturally, has it's own HTML header flow as well
- If the releases were displayed as HTML, the header flow will become illogical (ex. an H1 from the release coming after a H2 from the doc site.) See the comment below from the commit that changed them to text: 

> Export release notes as plain text to avoid illogical heading flow

_Originally posted by @thisisdano in https://github.com/uswds/uswds-site/pull/1227#discussion_r633158033_
             
A possible workaround: before injecting the HTML notes to the page, rewrite the markdown/HTML of all H-tags to be a few levels lower (probably 2). That works for the majority of the release notes. However, if a release note ever gets to H5 or lower (which they do is some cases, like 3.0.0-beta 4), we'd have to decided what the right markup would be when adding 2 levels goes beyond an H6. We could stop them at H6? Or convert to them to paragraphs? 

Thoughts @thisisdano @mejiaj?

**Comment by thisisdano** [comment]
I think I suggest that we remove this page entirely. Is there an important purpose to the page that is not covered by our changelogs or by our release documentation on Github?

**Comment by danbrady** [comment]
Yes, I believe all the information is already covered by the changelogs and release docs. Is there any value in checking out the popularity of this page in user analytics to see if its often visited that this issue should be fixed? Or should we simply update the content on the page to link over release page on Github?

**Comment by mejiaj** [comment]
Good suggestion @danbrady, I've checked analytics and this page doesn't get much traffic. No where near the most popular pages.
